data/sheragi/the emerald sword.txt

```diff
@@ -0,0 +1,854 @@
+			`	You go back to the same park as before with Albert and the three Hai in partial disguise, ice creams in hand and paws to commemorate the new acquisition.`
+			`	"That is one impressive ship," you mention.`
+			`	"True," Desyo says. "I've never seen a cannon that big before."`
+			choice
+				`	"That's what she said."`
+				`	"I wonder how it works."`
+					goto works
+			`	Desyo doesn't understand, but Eyama bursts into laughter. "Classic human joke. You see, it implies-"`
+			`	"God, don't explain it," Albert interrupts. "That was terrible. She's right though, that isn't something you see every day. Do you have any idea how it works?"`
+			label works
+			`	Rakei, who is more knowledgeable in this area, responds. "It's a fusion-pumped gamma-ray laser, or GRASER for short. The device is simple in concept, but the cost involved and the proportion of everything makes it impractical to develop, not to mention the way it works would probably be considered a war crime in many civilizations, thus making R&D not worth it. It's essentially using a nuclear flash to power a laser beam that penetrates and melts everything. Fascinating, but I'm hoping you'll only use it for peaceful purposes."`
+			`	Eyama adds, "I've been calling it the Dragonflame cannon. I think it's appropriate, even though it's technically not a flame in the chemical sense. The dragons of human legends could breathe fire, and although it's unlikely the Sheragi could actually do that, this cannon more than makes up for it."`
+			choice
+				`	"Do you think I could get in trouble by using it?"`
+					goto trouble
+				`	"It's not going to blow up my own ship, is it?"`
+			`	Rakei laughs. "Don't worry, the Sheragi did their homework. The simple materials they used means the gun can be maintained and repaired, so it continues to be structurally strong. Many of the components are active anyway, and they basically only exist for the short amount of time that the GRASER takes to fire."`
+				goto catch
+			label trouble
+			`	"I don't think you will, as long as you don't fire it into a planet. It doesn't cause unnecessary suffering like nerve gas, and it isn't as indiscriminate as a nuclear bomb. This is a focused beam. It's going to poke holes straight through any ship subject to it's wrath."`
+			label catch
+			`	"Then what is the catch?" you ask.`
+			`	"It eats a lot of power. I mean a lot. And it does that every shot. No single reactor will be capable of making it shoot as fast as it possibly could. Expect to invest a lot in batteries and reactors if you want to reoutfit that warship and still use the cannon.`
+			`	Desyo adds. "Also, don't go thinking you're invincible now. The shield matrix in that ship is incredibly primitive. Almost nonexistent. The hull is strong, but don't rely on it too much. Human technology could probably do better if they tried."`
```

Comment:
_Low key roasting human tech. Good stuff_

Reply:
lmao i swear this was unintentional, but i like it that way too.